native/cocos/platform/android/java/src/com/cocos/lib/CocosActivity.java

```diff
@@ -45,11 +46,14 @@
 
 public class CocosActivity extends GameActivity {
     private static final String _TAG = "CocosActivity";
+    private static final boolean _ENABLE_SUBSURFACE = false;
```

Comment:
Why add `_` prefix ?

Reply:
@bofeng-song

Reply:
Represents an internal variable

Reply:
It's already a variable in `private` access. NO need to add `_` prefix.
`_` prefix is not the same coding style as other `.java` file in our engine.

Reply:
_TAG also use the same coding style

Reply:
it is a constant variable, not a class member

Reply:
@freeboy 
Can you refer to the code of CocosHelper.java to repair it, delete the prefix of _TAG and _ENABLE_SUBSURFACE

Reply:
OK

> @freeboy Can you refer to the code of CocosHelper.java to repair it, delete the prefix of _TAG and _ENABLE_SUBSURFACE

OK